allure-docker-api-usage/send_results.ps1

```diff
@@ -0,0 +1,53 @@
+# Location of the report
+$ReportPath=".\allure-results-example"
+# Name of the project
+$ProjectId="default"
+# Url of allure reporting (without a trailing /)
+$BaseUrl = "http://localhost:5050"
+# Execution Name
+$ExecutionName ="test"
+# Exection From
+$ExecutionFrom ="local"
+
+$SendResultsEndpoint="$BaseUrl/send-results?project_id=$ProjectId"
+$GenerateReportEndpoint="$BaseUrl/generate-report?project_id=$ProjectId"
+
+$Request = @{
+  "results" = @()
+}
+
+$Files = (Get-ChildItem -File -Path $ReportPath)
+foreach ($Item in $Files) {    
+    $Content = Get-Content -Path $Item.FullName
+    if ($Content){
+        $Result = @{
+          "file_name"= $Item.name
+          "content_base64"= [Convert]::ToBase64String([IO.File]::ReadAllBytes($Item.FullName))
+        }
+        $Request.results += $Result;
+    }
+}
+
+$json = ConvertTo-Json -InputObject $Request
+
+$result = Invoke-WebRequest -Uri $SendResultsEndpoint -ContentType 'application/json' -Method POST -Body $json -UseBasicParsing
+
+$response = Invoke-RestMethod -Uri "$GenerateReportEndpoint&ExecutionName=$ExecutionName&execution_from=$ExecutionFrom&executionType=test"
```

Comment:
@lazytesting  You have a typo related to the name of parameters. If you don't fix that from your code you won't have the behavior expected. https://github.com/fescobar/allure-docker-service#customize-executors-configuration

Instead of
```
&ExecutionName=...
```
should be
```
&execution_name=...
```

You have the same issue with another parameter. It should be `execution_type` instead of `executionType=...`

I fixed that in the master branch
https://github.com/fescobar/allure-docker-service/blob/master/allure-docker-api-usage/send_results.ps1

Reply:
thanks for letting me know!